app/src/main/assets/extensions/cookies/manifest.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Nit: License?

Reply:
JSONs don't allow standard comments. Looked over in AC and none of the extension manifest files have license added, so I think we have to make exceptions for these files